core/build.gradle.kts

```diff
@@ -1,14 +1,19 @@
 plugins {
-    kotlin("jvm")
+    kotlin("multiplatform")
     kotlin("plugin.serialization") version libs.versions.kotlin.get()
     id(libs.plugins.mavenPublish.get().pluginId)
 }
 
-repositories {
-    mavenCentral()
-}
+kotlin {
+    applyDefaultHierarchyTemplate()
+
+    jvm()
 
-dependencies {
-    implementation(kotlin("stdlib"))
-    implementation(libs.kotlinx.serialization.json)
+    sourceSets {
+        commonMain.dependencies {
+            implementation(libs.okio)
+            implementation(kotlin("stdlib"))
+            implementation(libs.kotlinx.serialization.json)
+        }
+    }
```

Comment:
I think you can revert all these changes related to build.gradle.kts

Reply:
Done 👍